### ISSUE TITLE
Short methods are overlapped with strange "debug info"

### ISSUE THREAD

**Issue by wyhasany** [JoseLion/pretty-jupiter#81]
The short logs are overlapped in following way:
```
    ✔ 2.6 (748ms)0% EXECUTING [15s]
    ✔ 2.8 (776ms)0% EXECUTING [16s]
```
it seems like `0% EXECUTING [16s]` comes from some other logger. You can override it adding additional white spaces up to 50th column. Then printed output would be clearer.

**Comment by wyhasany** [comment]
@JoseLion I can prepare a commit if you agree to overlap the end of output with extra whitespaces up to 80 columns (standard terminal width). Just let me know wdyt :)

**Comment by wyhasany** [comment]
Hmm... It looks pretty the same with newest version:
```
  ✔ get  (868ms)90% EXECUTING [14s]
  ✔ get  (901ms)90% EXECUTING [17s]
<===========--> 90% EXECUTING [17s]
> :test > 6 tests completed
> :test > Executing test tech...propertyitems.PropertyItemsTest
```
It looks like the bar with progression left some `lifecycle` output which is normally overridden by the tests. Anyway I've created a PR #86 as there was an issue with configuring pretty-jupiter with `build.gradle.kts`. There is no issue when I call:
```
./gradlew test --console=plain
```
It seems that rich console somehow interact with logging.

**Comment by wyhasany** [comment]
Here you can find example project recreating this issue:
[example-logs.zip](https://github.com/JoseLion/pretty-jupiter/files/6448112/example-logs.zip)

**Comment by wyhasany** [comment]
Here you can find related docs, but I don't see anything helpful:
https://docs.gradle.org/current/userguide/command_line_interface.html#sec:command_line_logging

**Comment by JoseLion** [comment]
@wyhasany yes, it seems like this is more related to the console output rather than the plugin's logging. I guess that's why it works correctly with `--console=plain`. I usually add the `TERM=dumb` env variable in CI environments to avoid this kind of issue.

In any case, I'll give the project you shared a try and get back to you. Maybe I find something there. 🙂 

PS: Thanks for the PR for Kotlin interop 🎉

**Comment by JoseLion** [comment]
@wyhasany I tried the example you shared with me, and I think I found the issue. It seems you are using a `Thread.sleep(200)` in all your tests, so gradle assumes the process is idle and prints the progress to the output. If you reduce the sleep time to 50 (or less) you'll see the issue disapers (probably because Gradle has some threshold to consider a process idle).

IMO, this should not be an issue when running real tests. Even if a test is really expensive and takes an age to complete, it's not considered idle time since the thread is actually busy. 🙂

Let me know if this makes sense to you, or if you think this could still be an issue. I'm not sure if there's anything we can do from our side, but probably we can ask in https://discuss.gradle.org/ 😁

**Comment by wyhasany** [comment]
Hi! Thanks you a lot for your investigation. To be honest it's not an artificial issue as it looks like by my example 😅. Actually I see mixed logs in my integration tests. I think we can ask about it.

**Comment by wyhasany** [comment]
I've started a new [topic](https://discuss.gradle.org/t/rich-console-overlaps-custom-logs/40022) in https://discuss.gradle.org/

**Comment by JoseLion** [comment]
@wyhasany thanks for starting the topic! I've also seen this happening while running other kinds of tasks, so I'll be checking the thread too 🙂
